prow/plugins/bugzilla/bugzilla.go

```diff
@@ -199,7 +208,16 @@ func handleGenericComment(pc plugins.Agent, e github.GenericCommentEvent) error
 	return nil
 }
 
-func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
+func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) (err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			if rec, ok := r.(error); ok {
```

Comment:
Can this happen? I've never seen that before

Reply:
Changed it to just use `fmt.Errorf` without any casting. Casting the interface seems mainly to be useful if we specifically panic with our own type/struct (`encoding/json` does this for example), but we're not using any special types here.